plugins/mcreator-localization/lang/texts.properties

```diff
@@ -847,9 +847,14 @@ elementgui.common.enable=Check to enable
 elementgui.common.page_triggers=Triggers
 elementgui.common.page_visual=Visual
 elementgui.common.page_properties=Properties
+elementgui.common.page_inventory=Inventory
 elementgui.common.name_in_gui=Name in GUI:
-elementgui.common.event_right_clicked_air=When right clicked in air (player loc.))
-elementgui.common.event_right_clicked_block=When right clicked on block (hand loc.))
+elementgui.common.rarity=Rarity:
+elementgui.common.stack_size=Stack size:
+elementgui.common.item_model=<html>Item model:<br><small>Select the item model to be used. Supported: JSON, OBJ
+elementgui.common.event_right_clicked_air=When right clicked in air (player loc.)
+elementgui.common.event_right_clicked_block=When right clicked on block (hand loc.)

```

Comment:
Why did you move the lines for the right click event? And please copy/paste them well. You forgot a `)`
They should be that way :
`elementgui.common.event_right_clicked_air=When right clicked in air (player loc.)`
`elementgui.common.event_right_clicked_block=When right clicked on block (hand loc.)`